src/cluegetter/http.go

```diff
@@ -161,6 +161,8 @@ func httpHandlerMessageSearchEmail(w http.ResponseWriter, r *http.Request) {
 
 	rows, err := Rdbms.Query(`
 	SELECT m.id, m.date, CONCAT(m.sender_local, '@', m.sender_domain) sender, m.rcpt_count, m.verdict,
+           s.ip, s.reverse_dns, s.helo, s.sasl_username, s.sasl_method, s.cert_issuer, s.cert_subject,
+           s.cipher_bits, s.cipher, s.tls_version,
```

Comment:
What use case do you have in mind for selecting all these fields? I'm trying to think in what kind of scenario the tls version that was used could be useful. The same applies to some of the other fields.